pkgs/applications/networking/cluster/argocd-autopilot/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib, buildGo117Module, fetchFromGitHub }:
+
+buildGo117Module rec {
+  pname = "argocd-autopilot";
+  version = "0.3.0";
+  commit = "c8d17bef976649e4dc2428c14c39e30a0f846552";
+
+  src = fetchFromGitHub {
+    owner = "argoproj-labs";
+    repo = "argocd-autopilot";
+    rev = "v${version}";
+    sha256 = "sha256-tggE1T+oD/dJS9tD9xOExjhy+T1GDd0vwTerD3P2KvA=";
+  };
+
+  vendorSha256 = "sha256-UqZB3PknZrcgClZjpuFr9iLu1netwauTg/LpWoIVB0I=";
```

Comment:
for some reason, darwin doesn't like this. Generally this is from unicode being in file names or two file paths only differing by capitalizaiton.

```
installing
hash mismatch in fixed-output derivation '/nix/store/sfc82855k4qxig2lk20jrvfmmsc6mw6w-argocd-autopilot-0.3.0-go-modules':
  wanted: sha256:0hh72n15msgjhf9sphddfzbfw8pndghscqsn18hbfri7z7f439jj
  got:    sha256:13020nyz9jp3hghaziz9sqnvbk6spwm5xmxbhdjyv1sspj4xawsi
cannot build derivation '/nix/store/hxc62vf7k5l007pfs7y77g9wdyqfg114-argocd-autopilot-0.3.0.drv': 1 dependencies couldn't be built
error: build of '/nix/store/hxc62vf7k5l007pfs7y77g9wdyqfg114-argocd-autopilot-0.3.0.drv' failed
```

Reply:
> for some reason, darwin doesn't like this. Generally this is from unicode being in file names or two file paths only differing by capitalizaiton.
> 
> ```
> installing
> hash mismatch in fixed-output derivation '/nix/store/sfc82855k4qxig2lk20jrvfmmsc6mw6w-argocd-autopilot-0.3.0-go-modules':
>   wanted: sha256:0hh72n15msgjhf9sphddfzbfw8pndghscqsn18hbfri7z7f439jj
>   got:    sha256:13020nyz9jp3hghaziz9sqnvbk6spwm5xmxbhdjyv1sspj4xawsi
> cannot build derivation '/nix/store/hxc62vf7k5l007pfs7y77g9wdyqfg114-argocd-autopilot-0.3.0.drv': 1 dependencies couldn't be built
> error: build of '/nix/store/hxc62vf7k5l007pfs7y77g9wdyqfg114-argocd-autopilot-0.3.0.drv' failed
> ```

Any tip to track down the case?

Also TIL for MacOS...